<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1157</width>
    <height>764</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>PyGame Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGraphicsView" name="scene_preview">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="backgroundBrush">
           <brush brushstyle="NoBrush">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </property>
          <property name="foregroundBrush">
           <brush brushstyle="NoBrush">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </property>
          <property name="sceneRect">
           <rectf>
            <x>0.000000000000000</x>
            <y>0.000000000000000</y>
            <width>1024.000000000000000</width>
            <height>768.000000000000000</height>
           </rectf>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="dragMode">
           <enum>QGraphicsView::RubberBandDrag</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="scene_name_label">
          <property name="text">
           <string>Scene Name: None</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QListWidget" name="object_list">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="scene_list">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="0">
             <widget class="QPushButton" name="deletegameobject_button">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Delete GameObject</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="editgameobject_button">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Edit GameObject</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="newscene_button">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>New Scene</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="deleteallgameobjects_button">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Delete All of Type</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="editscenesettings_button">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Edit Scene Settings</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="newgameobject_button">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>New GameObject</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListWidget" name="object_heirarchy">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1157</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionNew_Scene"/>
    <addaction name="actionImport_Scene"/>
    <addaction name="actionNew_GameObject"/>
    <addaction name="actionImport_GameObject"/>
    <addaction name="actionSave_Scene"/>
    <addaction name="actionSave_Project"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionScene_Settings"/>
    <addaction name="actionProject_Settings"/>
    <addaction name="actionEditor_Settings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuBuild">
    <property name="title">
     <string>Build</string>
    </property>
    <addaction name="actionBuild"/>
    <addaction name="actionBuild_Settings"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionHeirarchy"/>
    <addaction name="actionObject_List"/>
    <addaction name="actionPreview"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuUI">
     <property name="title">
      <string>UI</string>
     </property>
     <addaction name="actionButton"/>
     <addaction name="actionText_Box"/>
     <addaction name="actionLabel"/>
    </widget>
    <widget class="QMenu" name="menuSound">
     <property name="title">
      <string>Sound</string>
     </property>
     <addaction name="actionSoundManager"/>
    </widget>
    <widget class="QMenu" name="menuNetwork">
     <property name="title">
      <string>Network</string>
     </property>
     <addaction name="actionNetworkManager"/>
    </widget>
    <addaction name="menuUI"/>
    <addaction name="menuSound"/>
    <addaction name="menuNetwork"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuBuild"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionNew_Scene">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>New Scene</string>
   </property>
  </action>
  <action name="actionNew_GameObject">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>New GameObject</string>
   </property>
  </action>
  <action name="actionSave_Scene">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Scene</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionScene_Settings">
   <property name="text">
    <string>Scene Settings</string>
   </property>
  </action>
  <action name="actionProject_Settings">
   <property name="text">
    <string>Project Settings</string>
   </property>
  </action>
  <action name="actionEditor_Settings">
   <property name="text">
    <string>Editor Settings</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionBuild">
   <property name="text">
    <string>Build...</string>
   </property>
  </action>
  <action name="actionBuild_Settings">
   <property name="text">
    <string>Build Settings</string>
   </property>
  </action>
  <action name="actionHeirarchy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Heirarchy</string>
   </property>
  </action>
  <action name="actionObject_List">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Object List</string>
   </property>
  </action>
  <action name="actionPreview">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
  </action>
  <action name="actionImport_GameObject">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Import GameObject</string>
   </property>
  </action>
  <action name="actionImport_Scene">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Import Scene</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionButton">
   <property name="text">
    <string>Button</string>
   </property>
  </action>
  <action name="actionText_Box">
   <property name="text">
    <string>Text Box</string>
   </property>
  </action>
  <action name="actionLabel">
   <property name="text">
    <string>Label</string>
   </property>
  </action>
  <action name="actionSoundManager">
   <property name="text">
    <string>SoundManager</string>
   </property>
  </action>
  <action name="actionNetworkManager">
   <property name="text">
    <string>NetworkManager</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
